renovate.json5

```diff
@@ -4,4 +4,25 @@
     ":semanticCommitTypeAll(fix)",
   ],
   dependencyDashboard: true,
+  argocd: {
```

Comment:
sweet, it's been on my todo list, does **/*install.yaml - **/*resrouces.yaml or **.yaml, or something like that work? might be a bit easier to just glob :)